ircDDBGateway/APRSTransmit/APRSTransmitAppD.cpp

```diff
@@ -79,8 +79,15 @@ int main(int argc, char** argv)
 	}
 
 	wxString repeater = parser.GetParam(0U);
+	if (repeater.length() != LONG_CALLSIGN_LENGTH) {
+		wxString callErrorMsg;
+		callErrorMsg << wxT("Invalid repeater call. ") << repeater << wxT("Call must be ") << LONG_CALLSIGN_LENGTH << wxT(" characters long.\n");
```

Comment:
Hi,  
  
It looks like I missed that one, Thanks for the hint. wxS shall be used as I understand the documentation.